empress_gui.py

```diff
@@ -767,34 +758,29 @@ def open_window_solution_space(self):
     def select_from_view_reconciliations_dropdown(self, event):
         """When "View reconciliations" dropdown is clicked."""
         if self.view_reconciliations_var.get() == "One MPR":
-            self.view_reconciliations_var.set("View reconciliations")
-            # Creates a new tkinter window
-            self.one_MPR_window = tk.Toplevel(self.master)
-            self.one_MPR_window.geometry("600x600")
-            self.one_MPR_window.title("One MPR")
-            # Bring the new tkinter window to the front
-            # https://stackoverflow.com/a/53644859/13698076
-            self.one_MPR_window.attributes('-topmost', True)
-            self.one_MPR_window.focus_force()
-            self.one_MPR_window.bind('<FocusIn>', self.bring_to_front)
-            ReconciliationsOneMPRWindow(self.one_MPR_window)
```

Comment:
Merge `ReconciliationsOneMPRWindow` class and `ReconciliationsOnePerClusterWindow` class into one `ReconciliationFrame` class.